README.md

```diff
@@ -15,10 +15,14 @@ by leveraging the modern web.
 if your database does not have a password your can use `postgres://username@localhost:portnumber/database_name`
 - Leave the `SECRET_KEY` as `this-is-hard-to-break`
 
-#### How to configure email
```

Comment:
Why are you getting rid of these lines?

Reply:
Thank you for the feedback. However this was caused by a rebase operation  but am going to fix it